app/blueprints/scoring/helpers.py

```diff
@@ -0,0 +1,19 @@
+import app.blueprints.scoring.forms.scores_and_justifications as scoring_form_classes
+from app.blueprints.services.data_services import get_scoring_system  # noqa
+from flask import abort
+from flask import current_app
+
+
+def get_scoring_class(round_id):
+    scoring_system = get_scoring_system(round_id)
+
+    try:
+        scoring_form_class = getattr(
+            scoring_form_classes, f"{scoring_system}ScoreForm"
+        )
+    except AttributeError:
+        current_app.logger.error(
+            f"Scoring system '{scoring_system}' not found."
+        )
+        abort(500, description="Scoring system not found.")
+    return scoring_form_class
```

Comment:
Hmm I'm not really fan of `getattr` & the `f-string` - could we maybe do a more simple factory or a mapping? Main reason for this suggestion is code traceability and less implicit functionality (i.e new `ScoreForm`s will need to follow the naming convention, which is unknown until you read the `getattr` bit of code) 

```python
if scoring_system == "ZeroToThree":
    return ZeroToThreeScoreForm
elif scoring_system == "OneToFive":
    return OneToFiveScoreForm
else:
    ... # log error
```

or 

```python
 mapping = {
    "ZeroToThree": ZeroToThreeScoreForm,
    "OneToFive": OneToFiveScoreForm
}
clazz = mapping.get(scoring_system)
if not clazz : ... # log error
```

Reply:
yeah i get what your saying, the original reason i did it this way is so we didnt have to maintainthis fail when we add a new scoring system but i agree, it is obfuscated in that code - i will make the change 👍

Reply:
pushed : )